esphome/components/bang_bang/bang_bang_climate.cpp

```diff
@@ -56,26 +56,27 @@ void BangBangClimate::control(const climate::ClimateCall &call) {
 }
 climate::ClimateTraits BangBangClimate::traits() {
   auto traits = climate::ClimateTraits();
-  traits.set_supports_current_temperature(true);
+  traits.add_feature_flags(climate::CLIMATE_SUPPORTS_CURRENT_TEMPERATURE |
+                           climate::CLIMATE_SUPPORTS_TWO_POINT_TARGET_TEMPERATURE | climate::CLIMATE_SUPPORTS_ACTION);
```

Comment:
Careful here -- bang_bang specifically _requires_ two points. 🙂
```suggestion
                           climate::CLIMATE_REQUIRES_TWO_POINT_TARGET_TEMPERATURE | climate::CLIMATE_SUPPORTS_ACTION);
```